pkg/pgmodel/querier/query_builder.go

```diff
@@ -242,7 +242,29 @@ func canAttemptPushdown(metadata *promqlMetadata) bool {
 	path := metadata.path // PromQL AST.
 	queryHints := metadata.queryHints
 	selectHints := metadata.selectHints
-	return extension.ExtensionIsInstalled && queryHints != nil && !hasSubquery(path) && selectHints != nil
+
+	// We can't push down without hints
```

Comment:
I'm sorry, I have to do this. :grin: 
```suggestion
	// We can't push down without hints.
```

Reply:
No problem! Fixed.